### DISCUSSION TITLE
Silly questions - Upgrade, Save, Restore ... didn't find answers

### DISCUSSION THREAD

**Discussion by Rege78** [flobz/psa_car_controller#671] [Q&A]
Hello

I'm using psa_car_controller with my e208 but I have some questions

app installed on a RPI4 with docker compose following the tutorial provided in Readme.md - works fine...

But ...
- How to check the installed version?
- How do I backup/restore datas ?
- How can I upgrade to latest with our starting from 0? (if not easy, it explains the previous question)

Thanks for answers!

**Comment by Dizzybacon** [comment]
On the second and third questions, it depends a bit on your set up. You should have a config directory in the pod that maps to a volume or is bound to a directory on your hard drive.

Provided that is the case you can upgrade to the latest without losing any of your config.
